### PR TITLE
Bug 1911307:  scale pods using the 'scale' subresource

### PR DESCRIPTION
**Pull request by rhamilto**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1911307

### CODE REVIEW

File: frontend/packages/console-shared/src/components/pod/PodRing.tsx [hunk 1]
```diff
@@ -54,7 +54,8 @@ const PodRing: React.FC<PodRingProps> = ({
   const handleScaling = _.debounce(
     (operation: number) => {
       const patch = [{ op: 'replace', path, value: operation }];
-      const promise: Promise<K8sResourceKind> = k8sPatch(resourceKind, obj, patch);
+      const opts = { path: 'scale' };
```
Comment (spadgett) [inline frontend/packages/console-shared/src/components/pod/PodRing.tsx:57]: You'll need to update the pod ring access check as well to use the `scale` subresource.

**Review by spadgett — COMMENTED** [review]
Thanks @rhamilto 👍

File: frontend/packages/console-shared/src/components/pod/PodRing.tsx (hunk 1 above)
Comment (rhamilto) [reply frontend/packages/console-shared/src/components/pod/PodRing.tsx:57]: This right?  https://github.com/openshift/console/pull/7807/commits/335306b2a232e861d5b88c83d9c6c9e033e88d59

Comment (spadgett) [reply frontend/packages/console-shared/src/components/pod/PodRing.tsx:57]: @rhamilto Looks right

**Review by spadgett — APPROVED** [review]
/lgtm

**Pull request merged**